subprojects/docs/src/docs/userguide/img/taskInputsOutputs.graffle

```diff
@@ -0,0 +1,816 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I saw the `.graphml` files were included in the directory alongside the PNGs, so assumed that the project files went there.